FoodFriends/app/src/main/java/com/example/foodfriends/fragments/ExploreFragment.java

```diff
@@ -101,46 +111,38 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             restaurantList = new ArrayList<Restaurant>();
             adapter = new ExploreAdapter(getContext(), restaurantList);
             rvRestaurants.setAdapter(adapter);
-            rvRestaurants.setLayoutManager(new LinearLayoutManager(getContext()));
+
```

Comment:
puttin recyclerview dependent classes inside a controller class makes it follow the SRP pattern more. https://en.wikipedia.org/wiki/Single-responsibility_principle

Where one class is responsible for a single function. Also for readability purposes :)